lib/Elastica/Scroll.php

```diff
@@ -135,6 +133,24 @@ public function rewind()
     }
 
     /**
+     * Cleares the search context on ES and marks this Scroll instance as finished.
+     */
+    public function clear()
+    {
+        if (null !== $this->_nextScrollId) {
+            $this->_search->getClient()->request(
```

Comment:
I wonder if there are reasons we should make this optional. Are there use cases where users don't want to remove the search context?

Reply:
Some lines below this one the scrollId is reset (`$this->_nextScrollId = null;`) and without that id the search context can't be used anymore. There is also no getter for the scrollId in the `Scroll` class. Therefore I don't see how the search context could still be useful afterwards.

Reply:
Make sense 👍